### PR TITLE
Update the support email address

### PR DESCRIPTION
**Pull request by ijdickinson**
FoodStandardsAgency/data-dot-food#106
Switch the support email address to use the new, shared contact point
for FSA support issues.

### CODE REVIEW

**Comment by ijdickinson** [comment]
@alexrcoley I've added the your help copy for FoodStandardsAgency/data-dot-food#106 to this PR

File: src/Help.vue [hunk 1]
```diff
@@ -1,10 +1,89 @@
 <template>
   <main>
-    <div class="container">
-      <h1>Help</h1>
-      <h2>1. About this page</h2>
-      <p>This page is currently a draft and contains temporary content.</p>
-      <p>In the future this will contain information about the website, likely a FAQ section, and a table of contents area.</p>
+    <div class="container readable-line-length">
```
Comment (joescottdave) [inline src/Help.vue:3]: Interesting class name, I like it

**Review by joescottdave — APPROVED** [review]
Lgtm

**Comment by alexrcoley** [comment]
LGTM. Thanks

**Pull request merged**